scala/scala.bzl

```diff
@@ -221,12 +245,13 @@ SourceJars: {srcjars}
     # _jdk added manually since _java doesn't currently setup runfiles
     # _scalac, as a java_binary, should already have it in its runfiles; however,
     # adding does ensure _java not orphaned if _scalac ever was not a java_binary
-    ins = (list(cjars) +
+    ins = (list(all_jars) +
```

Comment:
fixed. passing only transitive compile_time ijars.
passing only direct deps if dependency analysis is off.